lib/common.sh

```diff
@@ -93,10 +93,14 @@ FILESYSTEM=${FILESYSTEM:="/"}
 # BMO_RUN_LOCAL : run the Baremetal Operator locally (not in Kubernetes cluster)
 # CAPM3_RUN_LOCAL : run the Cluster API Provider Metal3 locally
 
-export CAPI_VERSION="${CAPI_VERSION:-"v1alpha4"}"
-export CAPM3_VERSION="${CAPM3_VERSION:-"v1alpha5"}"
 CAPM3_VERSION_LIST="v1alpha4 v1alpha5"
-if ! echo "${CAPM3_VERSION_LIST}" | grep -wq "${CAPM3_VERSION}"; then
+export CAPM3_VERSION="${CAPM3_VERSION:-"v1alpha5"}"
+
+if [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
+  export CAPI_VERSION="${CAPI_VERSION:-"v1alpha4"}"
+elif [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
+  export CAPI_VERSION="${CAPI_VERSION:-"v1alpha3"}"
+else
```

Comment:
@mboukhalfa  Sorry for noticing it late, but do you think we need this part?
```suggestion
if [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
  export CAPI_VERSION="v1alpha4"
elif [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
  export CAPI_VERSION=""v1alpha3"
else
```
Because, right now it is possible for someone to mess up the API versions. For example, with the current state, I exported `CAPI_VERSION=v1alpha3` and scripts will use `CAPM3_VERSION=v1alpha5` which is something we try to avoid.